.github/workflows/e2e-main.yaml

```diff
@@ -81,15 +81,18 @@ jobs:
 
       - name: Update podman
         run: |
-          # ubuntu version from kubic repository to install podman we need (v5)
-          ubuntu_version='23.04'
+          echo "ubuntu version from kubic repository to install podman we need (v5)"
+          ubuntu_version='24.04'
```

Comment:
Results in older podman version.

Reply:
@odockal  I have made the update, can you resolve this please?